modules/server/src/network/aggregations/AggregationAccumulator.ts

```diff
@@ -1,41 +1,33 @@
 import { SUPPORTED_AGGREGATIONS } from '../common';
 import { Aggregations, Bucket, NumericAggregations } from '../types/aggregations';
 import { AllAggregations } from '../types/types';
-import { RequestedFieldsMap } from '../util';
 
 type ResolveAggregationInput = {
 	aggregationsMap: AllAggregations;
-	requestedAggregationFields: string[];
 	accumulator: AllAggregations;
 };
 
+type AggregationsTuple = [Aggregations, Aggregations];
+type NumericAggregationsTuple = [NumericAggregations, NumericAggregations];
+
```

Comment:
change to tuple instead of loose array. resolving logic takes previous aggregation and current aggregation and returns a single aggregation